engine/src/main.cpp

```diff
@@ -305,27 +303,23 @@ int main(int argc, char *argv[]) {
     // Usually loaded from config.json
     std::string mission_name;
 
-    //this sets up the vegastrike config variable
-    setup_game_data();
-    //loads the configuration file .vegastrike/vegastrike.config from home dir if such exists
+    setup_game_data();  // TODO: Combine with vega_config::config settings object
     {
         std::pair<std::string, std::string> pair = ParseCommandLine(argc, argv);
         std::string subdir = pair.first;
         mission_name = pair.second;
 
         VS_LOG(info, (boost::format("GOT SUBDIR ARG = %1%") % subdir));
-        if (CONFIGFILE == 0) {
```

Comment:
If you're going to touch this C code, why not convert it to C++?

`const std::string config_file_name = "vegastrike.config";`

should work just fine. No?

Reply:
Yeah, probably. I'll leave that for another day though :-)